components/segment-bridge/base/rolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: sa-read-permissions
+  namespace: segment-bridge
```

Comment:
Not seeing namespace creation YAML for this namespace and we don't need our own namspace anyway, we can just put the account in the `toolchain-host-operator` namespace.

Reply:
fixed